plugins/link.py

```diff
@@ -31,6 +31,12 @@ def _process_links(self, links):
                 relink = source.get('relink', relink)
                 create = source.get('create', create)
                 path = source['path']
+            elif source == None:
+                basename = os.path.basename(destination)
+                if basename[0:1] == ".":
```

Comment:
Could you change the `"` to `'` (double quotes to single quotes)? That's the convention followed in the rest of the code.